modules/Bio/EnsEMBL/Compara/RunnableDB/PrepareMaster/UpdateGenomesFromMetadataFactory.pm

```diff
@@ -406,7 +406,7 @@ sub _parse_rename_rr_files {
             next if $_ !~ /[a-z]/;
             # the strain column can be blank, leading to mismatch of column numbers
             $_ = $_ . 'NULL' if $_ !~ /[a-z0-9]$/;
```

Comment:
As above, it should now be safe to remove these lines.

Reply:
🦺